scripts/yodaPlotNPCorr.py

```diff
@@ -9,6 +9,8 @@
 import matplotlib.pyplot as plt
 import numpy as np
 import yoda
+from fit import kafe2_fit as k2fit
+from fit import kafe2_hist_fit as k2hfit
```

Comment:
`kafe2_hist_fit` is commented out in `fit.py`. So, do you want to keep it? At least, comment this line out as well.